docs/installation.md

```diff
@@ -9,14 +9,15 @@ The Lychee gallery has a few system requirements. You will need to make sure you
     - MySQL _(version > 5.7.8)_ / MariaDB _(version > 10.2)_
 	- PostgreSQL _(version > 9.2)_
 	- Lychee's inbuilt SQLite3 support
-- PHP >= 8.0 with these PHP extensions:
+- PHP >= 7.4 with these PHP extensions:
     - BCMath
     - Ctype
     - Exif
     - Ffmpeg (optional &mdash; to generate video thumbnails)
     - Fileinfo
     - GD
     - Imagick (optional &mdash; to generate better thumbnails)
+    - Intl
```

Comment:
No this is no longer true. :)

Reply:
Got it. I've opened [#1163](https://github.com/LycheeOrg/Lychee/issues/1163) to address my installation issue.